AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Also seems like uncertainty is missing from this plot. Maybe a violin plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>See my previous comment about Bernoulli vs. Binomial: with the current model we will always only get 0s and 1s</p>

Reply:
<p>This new chart is way more intuitive. Nit. its posterior predictive count of plausible sign ups, not posterior</p><p> </p>